sys/sys.txt

```diff
@@ -543,6 +544,30 @@ stat {
 	pad2	int32
 }
 
+statx {
+	mask int32
+	blksize int32
+	attributes int64
+	nlink int32
+	uid int32
+	gid int32
+	mode int16
+	__spare0 int16
+	ino int64
+	size int64
+	blocks int64
+	__spare1 int64
+	atime statx_timestamp
+	btime statx_timestamp
+	ctime statx_timestamp
+	mtime statx_timestamp
+	rdev_major int32
+	rdev_minor int32
+	dev_major int32
+	dev_minor int32
+	__spare2 int64
```

Comment:
This should be `__spare2 array[int64, 14]` as in http://elixir.free-electrons.com/linux/latest/source/include/uapi/linux/stat.h#L98